### PR TITLE
AZP: add to CI gtest with proto enable

### PR DESCRIPTION
**Pull request by avildema**
add to CI gtest with proto enable eq yes

### CODE REVIEW

File: contrib/test_jenkins.sh [hunk 1]
```diff
@@ -1291,9 +1291,30 @@ run_tests() {
...
+
+	# all are running gtest
+	run_gtest "default"
+}
+
 prepare
 try_load_cuda_env
-if [ -n "$JENKINS_RUN_TESTS" ] || [ -n "$RUN_TESTS" ]
+if [[ "$PROTO_ENABLE" == "yes" ]]; then
+	run_test_proto_enable
```
Comment (yosefe) [inline contrib/test_jenkins.sh:1316]: we can still do check_machine

**Review by evgeny-leksikov — CHANGES_REQUESTED** [review]
CI has been failed with error. Can we report it as a warning?

**Review by evgeny-leksikov — APPROVED** [review]

File: contrib/test_jenkins.sh (hunk 1 above)
Comment (yosefe) [reply contrib/test_jenkins.sh:1317]:
```sh
if [ -n "$JENKINS_RUN_TESTS" ] || [ -n "$RUN_TESTS" ]
    check_machine
    if [[ "$PROTO_ENABLE" == "yes" ]]; then
        run_test_proto_enable
    else
        run_tests
    fi
fi
```

Comment (yosefe) [reply contrib/test_jenkins.sh:1317]: @avildema pls squash + force push due to small PR

**Review by yosefe — APPROVED** [review]

**Pull request merged**